### DISCUSSION TITLE
POSTAL v2 SMTP 535 Denied

### DISCUSSION THREAD

**Discussion by anthonyolazabal** [postalserver/postal#1582] [Help with using Postal] [answered]
Hello, 
I've just install the last version of Postal, setup a server and a domain (with DNS). 
After creating a SMTP credential a trying to add it on a wordpress, it fail showing :
250-My capabilities are
smtp_1      | [9] [2021-09-02T10:28:43.242] DEBUG -- : [RGLYGK] => 250 AUTH CRAM-MD5 PLAIN LOGIN
smtp_1      | [9] [2021-09-02T10:28:43.283] DEBUG -- : [RGLYGK] <= AUTH CRAM-MD5
smtp_1      | [9] [2021-09-02T10:28:43.284] DEBUG -- : [RGLYGK] => <HIDE INFO>
smtp_1      | [9] [2021-09-02T10:28:43.284] DEBUG -- : [RGLYGK] <= <HIDE INFO>
smtp_1      | [9] [2021-09-02T10:28:43.287] DEBUG -- : [RGLYGK] => 535 Denied
smtp_1      | [9] [2021-09-02T10:28:43.288] DEBUG -- : [RGLYGK] <= QUIT
smtp_1      | [9] [2021-09-02T10:28:43.288] DEBUG -- : [RGLYGK] => 221 Closing Connection

Do you have any idea with taht ?

**Comment by willpower232** [comment, marked as answer]
It implies that the details you're entering are incorrect. 

https://github.com/postalserver/postal/blob/2e7aaaba5a9a37f32bdff9a9fde0379834b2a4db/lib/postal/smtp_server/client.rb#L219-L244

On the credentials page there is a link to `Read more about sending outgoing e-mails`, make sure the details you have given wordpress match the ones on the page.

**Comment by anthonyolazabal** [comment]
Thanks for the quick answer, I figure how to do before your reply. The "login" to use when authenticating on SMTP is hidded behind a link and not directly accessible with the credential created, this was my mistake. 
Thanks
